### DISCUSSION TITLE
Does training resume or start from scratch each iteration?

### DISCUSSION THREAD

**Discussion by eringiglio** [talmolab/sleap#802] [General]
Hi! I'm running SLEAP on a computing cluster, so I've been making extensive use of the Colab training package features. However, I can't tell whether the automated training job outputs make use of previously trained models within the project (vs just training a new model from scratch each time using the predicted frames in the project). Is the unnamed model that is populated into the training job package the same as the versions that are selected in the training GUI?

**Comment by roomrys** [comment]
Hi @eringiglio,

Training will always start from scratch using the user-labeled frames that are currently in your SLEAP project. The training configuration files such as "training_config.json" retain the hyperparameters (see: Training Pipeline) used in training so that the user does not need to remember these. Whichever parameters are listed in the Training Pipeline when you select "Export training job package..." will be used for training. 
![image](https://user-images.githubusercontent.com/38435167/174852921-ba9a64c3-ecd7-481d-9459-0b23200a941f.png)
**Fig: The parameters seen in the Training Pipeline GUI are the parameters that will be used in the training package.**

_Note that only user-labeled frames will be used in training (to use predicted frames, the user will need to convert these predictions to user-labeled instances via a double click on the instance)._
